bin/index.js

```diff
@@ -57,9 +56,7 @@ module.exports = function(Aquifer, AquiferGitConfig) {
       return;
     }
 
-    var jsonPath        = path.join(Aquifer.projectDir, 'aquifer.json'),
-        json            = jsonFile.readFileSync(jsonPath),
-        make            = path.join(Aquifer.projectDir, json.paths.make),
+    var make            = path.join(Aquifer.projectDir, Aquifer.project.config.paths.make),
```

Comment:
I hate to be picky, but there is also a dynamically generated property on Aquifer that holds absolute paths to assets.

`path.join(Aquifer.projectDir, Aquifer.project.config.paths.make)`
can be
`Aquifer.project.absolutePaths.make`